manifests/01-operator-config.yaml

```diff
@@ -19,3 +19,7 @@ spec:
     - name: GettingStartedBanner
       visibility:
         state: Enabled
+    perspectives:
+    - id: dev
+      visibility:
+        state: Disabled
```

Comment:
Will this only apply to new installs? Since this is a create-only resource, clusters upgraded from previous versions won't have the change, correct?

Reply:
I think we need some logic in the operator to default the value to false for `dev` if there is no existing perspective config.

We'll need to be mindful of the edge case where the `admin` perspective is disabled and there is no config for the `dev` perspective. In that case, we shouldn't also disable `dev` by default. We don't want to disable both.

Reply:
Do we need to do anything for existing users who upgrade the cluster to 4.19 if no config is present? As we are not sure if they are using Developer perspective.

Reply:
@vikram-raj Yes. Clusters upgraded to 4.19 should have the dev perspective disabled by default unless there is specific perspective config. Admins who want the perspective can always reenable it through config.